core/common/src/BriefcaseTypes.ts

```diff
@@ -25,10 +25,16 @@ export enum DownloadBriefcaseStatus {
   Error,
 }
 
-/** Operations allowed when synchronizing changes between the Briefcase and iModelHub
+/** Whether a briefcase is editable or may only accept incoming changesets from iModelHub
  * @public
  */
-export enum SyncMode { FixedVersion = 1, PullAndPush = 2, PullOnly = 3 }
+export enum SyncMode {
+  FixedVersion = 1,

```

Comment:
undocumented.

Reply:
i tried adding @deprecated here, but gave up. I'll add a comment to explain what it was and how to migrate to the new approach.

Reply:
added